packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1074,38 +1077,51 @@ export class AccountHandler {
       return false;
     };
 
-    const skipTokenVerification = async (request: AuthRequest, account: any) => {
-      // Skip all checks to simulate an unverified session token state
-      if (this.config.signinConfirmation.tokenVerification === false) {
-        return false;
-      }
-
+    const skipTokenVerification = async (
+      request: AuthRequest,
+      account: any
+    ) => {
       // Check to see if there has been a recent, successfully-verified login from
       // the same device (as indicated by User-Agent string)
       if (this.config.signinConfirmation.deviceFingerprinting?.enabled) {
         try {
           const currentUserAgent = request.headers['user-agent'] || '';
-          const skipTimeframeMs = this.config.signinConfirmation.deviceFingerprinting.duration as unknown as number;
-
-          const isRecognized = await isRecognizedDevice(this.db, account.uid, currentUserAgent, skipTimeframeMs);
+          const skipTimeframeMs = this.config.signinConfirmation
```

Comment:
Only change I made in this function is removing that env var. The rest is just formatting 🙃